src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisitionExtensions.cs

```diff
@@ -19,5 +20,6 @@ namespace Microsoft.Identity.Web
     /// </summary>
     /// <param name="builder">Builder</param>
     /// <param name="acquireTokenOptions">Token acquisition options for the request. Can be null.</param>
-    public delegate void BeforeTokenAcquisitionForTestUser(AcquireTokenByUsernameAndPasswordConfidentialParameterBuilder builder, AcquireTokenOptions? acquireTokenOptions);
+    /// <param name="user">User claims.</param>
+    public delegate void BeforeTokenAcquisitionForTestUser(AcquireTokenByUsernameAndPasswordConfidentialParameterBuilder builder, AcquireTokenOptions? acquireTokenOptions, ClaimsPrincipal user);
```

Comment:
@neha-bhargava  Is this naming correct? Why does it have `Test` in it?

Reply:
already discussed last week, will follow up offline
JM signed off on the API as well.